app/serializers/user_serializer.rb

```diff
@@ -2,6 +2,7 @@ class UserSerializer
   include Serialization::PanoptesRestpack
   include RecentLinkSerializer
   include MediaLinksSerializer
+  include CachedSerializer
```

Comment:
This one will mean we don't get the correct uploaded_subjects count being serialized until the cache resets. If this is an issue i'll look at either touching the timestamp on the resource for the upload subject event or remove the cache.